JsonLd/ContextBuilder.php

```diff
@@ -59,6 +65,11 @@ public function __construct(
         $this->nameConverter = $nameConverter;
     }
 
+    public function setEventDispatcher(EventDispatcherInterface $eventDispatcher)
```

Comment:
As the event dispatcher is mandatory, you need to add it to the constructor (we still accept BC break until the 1.0.0 is released).